gradio/utils.py

```diff
@@ -165,11 +175,13 @@ def swap_blocks(self, demo: Blocks):
         demo.is_running = True
         demo.allowed_paths = self.running_app.blocks.allowed_paths
         demo.blocked_paths = self.running_app.blocks.blocked_paths
-        demo.theme = self.running_app.blocks.theme
-        demo.head_paths = self.running_app.blocks.head_paths
-        demo.css = self.running_app.blocks.css
-        demo.head = self.running_app.blocks.head
-        demo.css_paths = self.running_app.blocks.css_paths
+
+        demo.theme = self.get_attribute("theme", demo)
```

Comment:
For supporting hot reload of theme or css when its specified via the blocks constructor